packages/gatsby-theme-docs/src/components/video-player.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
+import { customProperties } from '@commercetools-uikit/design-system';
+import styled from '@emotion/styled';
+
+const VideoPlaceholder = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  aspect-ratio: 16 / 9;
+  height: ${(props) => props.height}px;
```

Comment:
is the height line a leftover from the previous implementation?

Reply:
Indeed 🤦